Libplanet/Tx/RawTransaction.cs

```diff
@@ -12,6 +12,8 @@ namespace Libplanet.Tx
 
         public static readonly byte[] SignerKey = { 0x73 }; // 's'
 
+        public static readonly byte[] GenesisHashKey = { 0x67 }; // 'g'
```

Comment:
This change should be added to the <q>Backward-incompatible storage format changes</q> section.